source/includes/_resource.videos.md

```diff
@@ -131,6 +131,10 @@ vhx.videos.create({
   "tracks": {
     "subtitles": []
   },
+  "advertising": {},
+  "metadata": {
+    "advertising_keywords": []
```

Comment:
we should also illustrate other metadata fields. other possible key / values are:

series_name: "Game of Thrones"
season_name: "Season 1"
season_number: 1
episode_number 5

OR 

movie_name: "The Martian"

probably more beneficial to outline the episode use-case and then in the description we can say if it's a movie, it'd be "movie_name".

this now complicates the internal=1 thing. if easier, let's just make it public.